src/toil/provisioners/gceProvisioner.py

```diff
@@ -828,7 +833,8 @@ def _waitForAppliance(cls, ip_address, role, keyName='core'):
         startTime = time.time()
         while True:
             if time.time() - startTime > cls.maxWaitTime:
-                logger.error("Appliance failed to start on machine with ip" % ip_address)
+                logger.error("Appliance failed to start on machine with ip %s" % ip_address)
+                logger.error("Check if the appliance is valid.")
```

Comment:
Maybe it would be helpful to elaborate a little more on what exactly `valid` means here.

Reply:
Such as?

Reply:
I think Jesse's suggesting telling the user what they can change to fix it.  I agree this error message is really general, like printing "Software did not work."